static/js/click_handlers.js

```diff
@@ -231,6 +231,26 @@ exports.initialize = function () {
         $(e.currentTarget).tooltip('destroy');
     });
 
+    // TOOLTIP FOR EMOJIS IN MESSAGES
+    $('#main_div').on('mouseenter', '#message-emojis', (e) => {
+        e.stopPropagation();
+        const elem = $(e.currentTarget);
+        const title = elem.attr('title');
+
+        elem.tooltip({
+            title: title,
+            trigger: 'hover',
+            placement: 'bottom',
```

Comment:
I think `top` would be a better placement because with `bottom` the tooltip can be partially covered by the mouse cursor.

Reply:
I tested this out with `top`, and it feels a bit funny.  Especially if the message is a one line emoji, as it obstructs the user name.

I also noticed only a very minor cover of the tooltip (which never covered the actual text) if you hover over the emoji coming up from the bottom.  Otherwise this a non-issue.

Reply:
I tested `top` initially, and I also think that `bottom` is better for this.